app/controllers/api/bounties_controller.rb

```diff
@@ -14,8 +14,20 @@ class Api::BountiesController < Api::ApiController
 =end
 
   def index
-    @product = Product.find_by!(slug: params[:org_id])
-    @bounties = @product.tasks.where.not(state: [:closed, :resolved]).order('priority ASC NULLS LAST').limit(3)
+    @product = Product.find_by(slug: params[:org_id])
+    if !@product
+      @product = Product.find_by(id: params[:org_id])
+    end
+    n = params[:limit]
+    if !n
+      n = 3
+    end
+    state = params[:state]
+    if !state
+      @bounties = @product.tasks.where.not(state: [:closed, :resolved]).order('priority ASC NULLS LAST').limit(n)
```

Comment:
Line is too long. [113/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.